Dockerfile

```diff
@@ -1,8 +1,18 @@
 # Builder
-
 FROM golang:1.12.6-alpine3.9 as builder
 
-WORKDIR /go/src/github.com/openbank/openbank
+RUN apk --no-cache add ca-certificates git
+
+WORKDIR /src/github.com/openbank/openbank
```

Comment:
Sorry, you didn't understand what I meant - I meant literally just `WORKDIR /src`. You don't need the GOPATH-like directory structure.

Reply:
ahh sorry,
my understanding is if we remove "go" directory, it will not considered as GOPATH
so , i will remove all the dir structure. so sorry